xml/_filter.xml

```diff
@@ -141,9 +141,12 @@
       <typeFilter name="*" expose="false" />
     </namespaceFilter>
     <namespaceFilter name="Microsoft.Extensions.Internal">
+      <typeFilter name="ISystemClock" expose="true" />
       <typeFilter name="*" expose="false" />
     </namespaceFilter>
     <namespaceFilter name="Microsoft.Extensions.Localization.Internal">
+      <typeFilter name="AssemblyWrapper" expose="true" />
```

Comment:
These both appear to be `internal`. I'm not sure if they should be exposed here. They are in the asp.net repo. So we should ask the ASP.NET owner.

[https://github.com/dotnet/aspnetcore/blob/1453c5bd2cf886f6846d2cc00b8106dbd0fdbd53/src/Localization/Localization/src/Internal/IResourceStringProvider.cs#L13](https://github.com/dotnet/aspnetcore/blob/1453c5bd2cf886f6846d2cc00b8106dbd0fdbd53/src/Localization/Localization/src/Internal/IResourceStringProvider.cs#L13)
[https://github.com/dotnet/aspnetcore/blob/1453c5bd2cf886f6846d2cc00b8106dbd0fdbd53/src/Localization/Localization/src/Internal/AssemblyWrapper.cs#L14](https://github.com/dotnet/aspnetcore/blob/1453c5bd2cf886f6846d2cc00b8106dbd0fdbd53/src/Localization/Localization/src/Internal/AssemblyWrapper.cs#L14)

cc @ryanbrandenburg

Reply:
Both IResourceStringProvider and AssemblyWrapper are internal  to avoid public usage, and therefore shouldn't appear in the API documentation.

Reply:
Gotcha. They are used as parameter types in [ResourceManagerStringLocalizer constructors](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.localization.resourcemanagerstringlocalizer.-ctor?view=dotnet-plat-ext-3.1#overloads), but those are annotated as "intended for testing purposes only".